src/main/java/dev/jbang/DependencyUtil.java

```diff
@@ -131,26 +137,62 @@ public ModularClassPath resolveDependencies(List<String> deps, List<MavenRepo> r
 			mavenRepo.apply(resolver);
 		});
 
+		MavenWorkingSession xyz = ((MavenWorkingSessionContainer) resolver).getMavenWorkingSession();
```

Comment:
some side effect here? Because it seems xyz is not being used.

Reply:
whoop nope - leftover from me exploring to use raw maven. turned out not needed for now.